hw02_unpack_string/unpack_test.go

```diff
@@ -16,6 +16,10 @@ func TestUnpack(t *testing.T) {
 		{input: "abccd", expected: "abccd"},
 		{input: "", expected: ""},
 		{input: "aaa0b", expected: "aab"},
+		{input: "aaa0b", expected: "aab"},
+		{input: `a`, expected: `a`},
+		{input: `a2c`, expected: `aac`},
+		{input: `bht3r`, expected: `bhtttr`},
 		// uncomment if task with asterisk completed
```

Comment:
```suggestion
{input: `тес3тКириллиц2ы2`, expected: `тессстКириллиццыы`},
		// uncomment if task with asterisk completed
```
Добрый день, у меня валится вот этот тест, хотя должен работать. Посмотрите что не так?

Reply:
Переписал решение полностью. Была допущена ошибка с чтением строк. Теперь тесты проходят

Reply:
Да, вижу, так намного лучше. ДЗ принял.